cubi_tk/sodar/pull_raw_data.py

```diff
@@ -155,7 +155,7 @@ def execute(self) -> typing.Optional[int]:
         if not out_path.exists():
             out_path.mkdir(parents=True)
 
-        investigation = api.investigations.get(
+        investigation = api.samplesheet.retrieve(
```

Comment:
How did this affect the other sample sheet type?

@holtgrewe, care to comment? I'm not familiar with SODAR API syntax...